app/Http/Controllers/Auth/AuthController.php

```diff
@@ -58,86 +54,10 @@ public function add(Request $request)
                 Token::generate('email-activate', $this->email_activation_token_time, $user->id),
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "email-activate" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=php%3AS1192)